source/common/stats/allocator_impl.cc

```diff
@@ -362,7 +366,7 @@ void AllocatorImpl::markCounterForDeletion(StatName name) {
   if (iter == counters_.end()) {
     return;
   }
-  ASSERT(!hasStat(deleted_counters_, *iter));
+  // Duplicates are checked in ~AllocatorImpl.
```

Comment:
s/checked/ASSERTed/ to make it clearer that doesn't happen in production.

Reply:
Done